src/main/java/com/google/cloud/resourcemanager/OrgPolicyInfo.java

```diff
@@ -0,0 +1,398 @@
+          return orgPolicyInfo.toProtobuf();
+        }
+      };
+
+  private BoolPolicy boolPolicy;
+  private String constraint;
+  private String etag;
+  private Policies policies;
+  private RestoreDefault restoreDefault;
+  private String updateTime;
+  private Integer version;
+
+  /** Used For boolean Constraints, whether to enforce the Constraint or not. */
+  static class BoolPolicy implements Serializable {
+
+    private static final long serialVersionUID = -2133042982786959351L;
+    private final Boolean enforce;
+
+    BoolPolicy(Boolean enforce) {
+      this.enforce = enforce;
+    }
+
+    public boolean getEnforce() {
+      return enforce;
+    }
+
+    @Override
+    public String toString() {
+      return MoreObjects.toStringHelper(this).add("enforce", getEnforce()).toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+      if (this == o) {
+        return true;
+      }
+      if (o == null || getClass() != o.getClass()) {
+        return false;
+      }
+      BoolPolicy that = (BoolPolicy) o;
+      return Objects.equals(enforce, that.enforce);
+    }
+
+    @Override
+    public int hashCode() {
+      return Objects.hash(enforce);
+    }
+
+    BooleanPolicy toProtobuf() {
+      return new BooleanPolicy().setEnforced(enforce);
+    }
+
+    static BoolPolicy fromProtobuf(BooleanPolicy booleanPolicy) {
+      return new BoolPolicy(booleanPolicy.getEnforced());
+    }
+  }
+
+  /**
+   * The organization ListPolicy object.
+   *
+   * <p>ListPolicy can define specific values and subtrees of Cloud Resource Manager resource
+   * hierarchy (Organizations, Folders, Projects) that are allowed or denied by setting the
+   * allowedValues and deniedValues fields. This is achieved by using the under: and optional is:
+   * prefixes. The under: prefix denotes resource subtree values. The is: prefix is used to denote
+   * specific values, and is required only if the value contains a ":". Values prefixed with "is:"
+   * are treated the same as values with no prefix. Ancestry subtrees must be in one of the
+   * following formats: - "projects/", e.g. "projects/tokyo-rain-123" - "folders/", e.g.
+   * "folders/1234" - "organizations/", e.g. "organizations/1234" The supportsUnder field of the
+   * associated Constraint defines whether ancestry prefixes can be used. You can set allowedValues
+   * and deniedValues in the same Policy if allValues is ALL_VALUES_UNSPECIFIED. ALLOW or DENY are
+   * used to allow or deny all values. If allValues is set to either ALLOW or DENY, allowedValues
+   * and deniedValues must be unset.
+   */
+  static class Policies implements Serializable {
+
+    private static final long serialVersionUID = -2133042982786959352L;
+
+    private final String allValues;
+    private final List<String> allowedValues;
+    private final List<java.lang.String> deniedValues;
+    private final Boolean inheritFromParent;
+    private final String suggestedValue;
+
+    public Policies(
```

Comment:
class isn't public, does the constructor need to be?

Reply:
done